src/platform/qt/SettingsView.cpp

```diff
@@ -710,6 +711,7 @@ void SettingsView::reloadConfig() {
 	loadSetting("gba.forceGbp", m_ui.forceGbp);
 	loadSetting("vbaBugCompat", m_ui.vbaBugCompat, true);
 	loadSetting("updateAutoCheck", m_ui.updateAutoCheck);
+	loadSetting("showFilenameInLibrary", m_ui.showFilenameInLibrary, false);
```

Comment:
```suggestion
	loadSetting("showFilenameInLibrary", m_ui.showFilenameInLibrary);
```
`false` is the default so you don't need to pass it.